settings.py

```diff
@@ -221,7 +224,7 @@ def get_from_env(var, default):
 # authentication systems enabled
 # AUTH_ENABLED_SYSTEMS = ['password','facebook','twitter', 'google', 'yahoo']
 AUTH_ENABLED_SYSTEMS = get_from_env('AUTH_ENABLED_SYSTEMS',
-                                    get_from_env('AUTH_ENABLED_AUTH_SYSTEMS', 'password,google,facebook')
+                                    get_from_env('AUTH_ENABLED_AUTH_SYSTEMS', 'password,google,facebook,ldap')
```

Comment:
I don't think we should enable LDAP by default, can we remove that here?

Reply:
@benadida you're right! It shouldn't, so I've just removed it. Thanks for the reply!